crowbar_framework/lib/crowbar/conduit_resolver.rb

```diff
@@ -106,7 +106,7 @@ def if_speed_remap
 
         sorted_ifs.each do |intf|
           speeds = cr_ohai_network[intf]["speeds"]
-          speeds = ["1g"] unless speeds # legacy object support
+          speeds = ["1g"] if speeds.nil? || speeds.empty? # legacy object support
```

Comment:
You can use `speeds.blank?`  here. 

http://api.rubyonrails.org/v4.2/classes/Object.html#method-i-blank-3F